### ISSUE TITLE
Automatically perform geolocation check when saving Quick Edits

### ISSUE THREAD

**Issue by tempsaint** [bmlt-enabled/wordpress-bmlt-workflow#165]
Currently, if I open the Quick Edit screen and make a slight change and Save, or even don't change anything but click Save instead of Cancel, the plugin gives a geolocation error if I try to approve the submission. So, it looks like if any "save" function is performed in the Quick Edit, it requires a Geolocation check. 

I'd like to propose that clicking Save in Quick Edit automatically perform the Geolocation check (as well as Saving) so that a two step process isn't required by admins.

**Comment by Klodd65** [comment]
Indeed, i try to correct a county with Quick Edit, but whether or not I click Save, it does not save.

**Comment by tempsaint** [comment]
@nigel-bmlt - is this something that got fixed in the latest version. I know you were working on the geocoding stuff.

**Comment by nigel-bmlt** [comment]
fixed in 1.1.40
